detox/android/detox/src/full/java/com/wix/detox/reactnative/ReactNativeExtension.kt

```diff
@@ -39,7 +39,7 @@ object ReactNativeExtension {
             reloadReactNativeInBackground(it)
             val reactContext = awaitNewReactNativeContext(it, previousReactContext)
 
-            setupIdlingResources(reactContext, networkSyncEnabled)
+            setSynchronizationState(reactContext, networkSyncEnabled)
```

Comment:
There's definitely a naming issue here. What does "state" mean? It's not descriptive enough.